### PR TITLE
avatar component added

### PR DESCRIPTION
**Pull request by shivanigangadharan**

### CODE REVIEW

File: components/avatar.css [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    display: flex;
+    align-items: center;
+    gap: 30px;
+}
+.avatar-img{
+    border-radius: 999px;
+}
+.avatar-icon{
+    padding: 1.5rem;
+    background: lightgrey;
```
Comment (rutvikpumak) [inline components/avatar.css:12]: use color variable

File: pages/docs/docs.html [hunk 2]
```diff
@@ -52,6 +52,33 @@
...
+            <div class="component">
+                <text class="title"> Avatar</text>
+                <text class="subtext"> An avatar is typically used to display a user profile as a picture, an icon, or short text.</text>
+                <text class="example"> Example</text>
+
+                <!-- ### Avatar component -->
+                <div class="container-avatar">
+
+                    <!--Avatar using image-->
+                    <img class="avatar-img" src="https://picsum.photos/id/1025/90/90">
```
Comment (MaruthiPothuganti) [inline pages/docs/docs.html]: Good to use Alternate attributes for accessibility for images as a better practice

**Review by MaruthiPothuganti — COMMENTED** [review]
Suggested changes for Accesbility

File: pages/docs/docs.html (hunk 2 above)
Comment (shivanigangadharan) [reply pages/docs/docs.html]: Added the alternative attribute in commit 0202d6e.

File: components/avatar.css (hunk 1 above)
Comment (shivanigangadharan) [reply components/avatar.css]: Done in commit 606c658.

File: components/avatar.css [hunk 3]
```diff
@@ -0,0 +1,23 @@
+.container-avatar{
+    margin: 2.5rem 0px 0px 0px;
+    display: flex;
+    align-items: center;
+    gap: 30px;
+}
+.avatar-img{
+    border-radius: 999px;
```
[the remark this answers is not in the document]
Comment (shivanigangadharan) [reply components/avatar.css]: Done 606c658.

**Pull request merged**